content/en/apps/examples/covid-rdt-reference-app.md

```diff
@@ -0,0 +1,490 @@
+
+## Problem Being Addressed
+
+The original call for proposals best describes why Medic created this app: 
+
+> FIND is working towards supporting countries in implementing an effective test-trace-isolate response using digital tools. To this end, FIND is looking to partner with leading digital solution providers to accelerate the development and deployment of a set of minimum functionalities for the collection of COVID RDT data and supporting incorporation of these functionalities into existing digital tools for use in low- and middle-income country settings.
+
+## Solution Overview
+
+This reference app provides a base layer of functionality that you can easily customize to meet the needs of your health program.  Specifically, this application has examples of:
+ * Provisioning COVID-19 RDT tests 
+ * Capturing COVID-19 RDT results
+ * Storing all RDT data, including pictures, in the CHT
+ * Best practices of Health Facility, CHW, patient and RDT hierarchy in the CHT
+
+## Workflow
+
+There are three main components to this application:
+ * The CHT forms to provision and capture RDTs (Green)
+ * Using a third-party app to use an RDT (Yellow)
+ * Tasks to remind CHWs to complete an RDT that has been started (Blue) 
+
+{{< figure src="flow.png" link="flow.png" class="right col-12 col-lg-12" >}}
+
+## Training Materials
+
+Medic is providing images and videos for use in training CHWs on how to use the CHT with the RD-Toolkit. These could be combined with existing in [app training]({{< ref "apps/examples/covid-education" >}}) if needed. As well, the RD-Toolkit as in app instructions for how to use specific RDTs.  
+
+### Images
+
+27 high resolutions images taken from a demonstration CHW device are [available for download](CHT.COVID-19.RDT.Images.zip). These cover the entire usage of the reference app.  
+
+### Videos
+
+Provided here are two videos of the Provision and Capture forms in the COVID-19 application.
+
+#### Provision 
+
+This first video shows the left side of the workflow above to provision an RDT for a patient.  The CHW is shown finding Jessica Whitehouse's contact and choosing a new action of "RDT Covid-19 - Provision".  The CHW then does pre-test set up, checking Jessica's symptoms and location and confirming the test method and lot information. You can see the CHW launching the RD-Toolkit, reading the instructions and then, starting the session and timer.  With the RD-Toolkit sending back all the information to the CHT, a task "Capture Covid-19 RDT" can be seen in the CHT for the CHW to follow up on when the RD-Timer has completed. The last part of the video shows the completed provision report in the CHT:
+
+<!-- 
+  use raw HTML for now as hugo youtube short codes show 16:9 ration 
+  box instead of mobile portrait ~9:16
+-->
+<iframe id="ytplayer" type="text/html" width="290" height="610" src="https://www.youtube.com/embed/PGC0Mp6KtU4" frameborder="0"></iframe>
```

Comment:
Agreed! And I was trying to do the[ "cursor" in `scrcpy`](https://en.wikipedia.org/wiki/Scrcpy) - but of course you [can turn it on in android natively](https://www.skyneel.com/show-touches-in-android-phone).  I'll look into this!  As well,I'd be happy to do some edits to have "stages" and pause the video to show a interstitial white text on black background "CHW will now select the patient", "RDT test will not be provisioned including a timer and storing the session ID In the CHT" or some such thing.

Otherwise, if there's an in-house CHT demo video expert, I'd be happy to work with them!